lib/modules/platform/github/index.ts

```diff
@@ -166,14 +170,26 @@ export async function initPlatform({
     }
   }
   logger.debug({ platformConfig, renovateUsername }, 'Platform config');
-  const platformResult: PlatformResult = {
+  const gitHubRule: HostRule[] = [];
```

Comment:
```suggestion
  const hostRules: HostRule[] = [];
```

Assuming this variable isn't already declared elsewhere

Reply:
it is used elsewhere